### PR TITLE
Update SDK to 8.0.100-rc.1.23370.3

### PR DESCRIPTION
**Pull request by MackinnonBuck**

### CODE REVIEW

**Comment by AndriySvyryd** [comment]
Superseded by #31178

**Comment by bricelam** [comment]
Re-opening. We took this out of #31178 to unblock it

**Comment by bricelam** [comment]
Still probably blocked tho waiting for https://github.com/dotnet/roslyn/pull/69010 to propagate

File: global.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23354.1",
+    "version": "8.0.100-preview.7.23364.32",
```
Comment (AndriySvyryd) [inline global.json:3]:
```suggestion
    "version": "8.0.100-preview.7.23368.9",
```

File: global.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
     "allowPrerelease": true,
     "rollForward": "latestMajor"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23354.1",
+    "dotnet": "8.0.100-preview.7.23364.32",
```
Comment (AndriySvyryd) [inline global.json:8]:
```suggestion
    "dotnet": "8.0.100-preview.7.23368.9",
```

**Review by AndriySvyryd — APPROVED** [review]

**Comment by RussKie** [comment]
The MacOS build leg is failing....

File: global.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23354.1",
+    "version": "8.0.100-preview.7.23368.9",
```
Comment (AndriySvyryd) [inline global.json:3]:
```suggestion
    "version": "8.0.100-preview.7.23369.1",
```

File: global.json [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
     "allowPrerelease": true,
     "rollForward": "latestMajor"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23354.1",
+    "dotnet": "8.0.100-preview.7.23368.9",
```
Comment (AndriySvyryd) [inline global.json:8]:
```suggestion
    "dotnet": "8.0.100-preview.7.23369.1",
```

File: global.json [hunk 5]
```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23354.1",
+    "version": "8.0.100-preview.7.23369.1",
```
Comment (AndriySvyryd) [inline global.json:3]:
```suggestion
    "version": "8.0.100-preview.7.23370.3",
```

File: global.json [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
     "allowPrerelease": true,
     "rollForward": "latestMajor"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23354.1",
+    "dotnet": "8.0.100-preview.7.23369.1",
```
Comment (AndriySvyryd) [inline global.json:8]:
```suggestion
    "dotnet": "8.0.100-preview.7.23370.3",
```

File: global.json [hunk 7]
```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23354.1",
+    "version": "8.0.100-preview.7.23370.3",
```
Comment (AndriySvyryd) [inline global.json:3]:
```suggestion
    "version": "8.0.100-rc.1.23370.3",
```

File: global.json [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
     "allowPrerelease": true,
     "rollForward": "latestMajor"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23354.1",
+    "dotnet": "8.0.100-preview.7.23370.3",
```
Comment (AndriySvyryd) [inline global.json:8]:
```suggestion
    "dotnet": "8.0.100-rc.1.23370.3",
```

**Pull request merged**